Gopkg.toml

```diff
@@ -0,0 +1,74 @@
+# Gopkg.toml example
```

Comment:
Please drop the commented code.

Reply:
Go dep generates a small example on top of the file to show the file's structure. I've left the file as it was generated, so that someone who will edit it knows what the structure is.

I will remove it

Reply:
I'm aware of that, nevertheless I'd prefer if there's no commented code snippets. Thanks :)